spec/operations/mailers/user_info_changed.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe Mailers::UserInfoChanged do
```

Comment:
this filename should change to ` spec/operations/mailers/user_info_changed_spec.rb `